Add minimum Spring Boot version to docs. (#2482)
getsentry/sentry-docs: src/platforms/java/guides/spring-boot/index.mdx

- automatically setting the release on `SentryEvent` when [Spring Boot Git integration is configured](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-git-info)
- automatically registering `BeforeSendCallback`, `BeforeBreadcrumbCallback`, `EventProcessor`, `Integration` beans

<Note><markdown>

> We support Spring Boot 2.1.0 or newer. Still on an older version? You can use [our legacy SDK](/platforms/java/legacy/spring/).

Reviewer: ```suggestion
We support Spring Boot 2.1.0 and above. Still on an older version? You can use [our legacy SDK](/platforms/java/legacy/spring/).
```

- automatically setting the release on `SentryEvent` when [Spring Boot Git integration is configured](https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-git-info)
- automatically registering `BeforeSendCallback`, `BeforeBreadcrumbCallback`, `EventProcessor`, `Integration` beans

> We support Spring Boot 2.1.0 or newer.

Reviewer 2: ```suggestion
<Note><markdown>

We support Spring Boot 2.1.0 or newer. Still on an older version? You can use [our previous version](/platforms/java/legacy/spring/).

</markdown></Note>
```
